lister/Program.cs

```diff
@@ -10,13 +10,24 @@ class Program
     {
         static void Main(string[] args)
         {
+            // The "@" means the following is a "verbatim" string literal. Normal
+            // string literals use the backslash (\) as an escape character. For example,
+            // you can include a tab in your string by adding "\t". Or you can include a
+            // double quote in your string using "\"".
+            //
+            // In a normal string literal if you need a backslash, you have to escape it
+            // like so: "\\". As you can see that can make file paths in strings really
+            // ugly. Thus C# provides the verbatim string literal.
+            //
+            // https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/string
+            //
             string path = @"C:\gitstuff\lister\data\simple-3-list.txt"; //no escape char because of @...why?
 
-            Console.WriteLine("Import data from the simple-3-list.txt file in the data directory"); 
```

Comment:
Sorry, I missed the whitespace changes in this commit. Normally I'd remove them all in a separate commit.